webclient/app/src/app/commons/appTemplateCard/AppTemplateCard.jsx

```diff
@@ -0,0 +1,128 @@
+import js from 'react-syntax-highlighter/dist/esm/languages/hljs/javascript';
+import dark from 'react-syntax-highlighter/dist/esm/styles/hljs/dark';
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+import AppTemplateDialog from "../../commons/appTemplateDialog/AppTemplateDialog";
+import AppTemplateRest from "../../services/AppTemplateRest"
+import ConfirmationDialog from "../alert/ConfirmationDialog";
+
+const ExpandMore = styled((props) => {
+    const { expand, ...other } = props;
+    return <IconButton {...other} />;
+})(({ theme, expand }) => ({
+    transform: !expand ? 'rotate(0deg)' : 'rotate(180deg)',
+    marginLeft: 'auto',
+    transition: theme.transitions.create('transform', {
+        duration: theme.transitions.duration.shortest,
+    }),
+}));
+
+
+function AppTemplateCard(props) {
+
+    SyntaxHighlighter.registerLanguage('javascript', js);
+
+    const { appTemplate, handleRefresh } = props;
+    const appTemplateCardStyles = AppTemplateCardStyles();
+    const { t } = useTranslation();
+    const [expanded, setExpanded] = React.useState(false);
+    const [selectedAppTemplate, setSelectedAppTemplate] = useState(false);
+    const [openDialog, setOpenDialog] = React.useState(false);
+    const appTemplateRest = new AppTemplateRest();
+
+    const [openDeleteDialog, setOpenDeleteDialog] = useState(false);
+    const deleteDialolgContent = ({ "title": "appTemplateDeleteDialog.title", "message": "appTemplateDeleteDialog.message" });
+
+    const closeDeleteDialog = () => {
+        setOpenDeleteDialog(false);
+    }
+
+    const handleExpandClick = () => {
+        setExpanded(!expanded);
+    };
+
+    const handleDialogOpen = () => {
+        setOpenDialog(true);
+        setSelectedAppTemplate(appTemplate);
+    };
+
+    const handleDialogClose = () => {
+        setOpenDialog(false);
+    };
+
+    const  handleDelete = (appTemplateId) => {
+        appTemplateRest.delete(appTemplateId).then(response => {
+            handleRefresh();
+        });
+    }
+
+    return (
+        <Container>
+            <Card color={"error"}>
+                <CardContent className={appTemplateCardStyles.CardContent}>
+                    <Grid container spacing={0}>
+                        <Grid item xs={7}>
+                            <Typography gutterBottom variant="h5" component="span">
+                                {appTemplate.name}
+                            </Typography>
+                        </Grid>
+                        <Grid item xs={5} align="right">
+                            <IconButton onClick={() => handleDialogOpen()}><Edit /></IconButton>
+                            <IconButton onClick={() => setOpenDeleteDialog(true)}><Delete /></IconButton>
+                        </Grid>
+                    </Grid>
+                    <Divider />
+                    <Typography variant="body2" color="text.secondary">
+                        <br />{appTemplate.description}
```

Comment:
A Padding would be more suitable than a ``<br/>``

Reply:
yes - it would be great to have an appropriate padding in the general stypesheet.

Reply:
not in this ticket...